homepage/index.php

```diff
@@ -75,16 +75,17 @@
             </div>
           </div>
           <div class="col-md-8">
-            <form class="form-inline form-give-home-signin mt">
+            <form class="form-inline form-give-home-signin mt pb border-bottom-small text-center">
               <div class="form-group">
-                <label for="exampleInputName2">Name</label>
-                <input type="text" class="form-control" id="exampleInputName2" placeholder="Jane Doe">
+                <span class="btn btn-default form-heading">EXISTING USERS</span>
```

Comment:
Generally write text on page as Proper casing. Solve the casing via CSS (text-transform).